skimage/feature/peak.py

```diff
@@ -11,8 +11,8 @@ def peak_local_max(image, min_distance=1, threshold_abs=None,
     Peaks are the local maxima in a region of `2 * min_distance + 1`
     (i.e. peaks are separated by at least `min_distance`).
 
-    If peaks are flat (i.e. multiple adjacent pixels have identical
-    intensities), the coordinates of all such pixels are returned.
+    If multiple peaks in the specified region have identical intensities,
+    the coordinates of all such pixels are returned.
```

Comment:
Yes, see https://github.com/scikit-image/scikit-image/issues/1831#issuecomment-180746693 for visual example.
If some pixel is detected as a peak in the `(2 * min_distance+1)x(2 * min_distance+1)` region, _not only adjacent_ pixels with the same intensity are also marked as peaks, but rather all of them.